src/index.html

```diff
@@ -2,20 +2,72 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div class="card__header">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac image"
+          class="card__image"
+        >
+      </div>
+      <div class="card__section1">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__description">
+          Product code: 195434
+        </p>
+      </div>
+      <div class="card__section2">
```

Comment:
```suggestion
      <div class="card__rating">
```
It's my suggestion. You can name them whatever you want, just please use descriptive names.